Assets/Scripts/PlayerController.cs

```diff
@@ -1,38 +1,54 @@
+using System;
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 public class PlayerController : MonoBehaviour
 {
-    public float rotationSpeed = 360;
-    public float acceleration = 100;
+    public Rigidbody2D playerRb;
+    public Camera camera;
+    public Transform rightFirePoint;
+    public Transform leftFirePoint;
     public GameObject bulletPrefab;
 
-    private Rigidbody2D playerRB;
-
     // Start is called before the first frame update
     void Start()
     {
-        playerRB = gameObject.GetComponent<Rigidbody2D>();
+        playerRb = gameObject.GetComponent<Rigidbody2D>();
     }
 
+    private Vector2 movement;
+    private Vector2 mousePos;
+    
+    private float moveSpeed = 10f;
```

Comment:
`moveSpeed` on the other hand could be public, for easier modification in the editor.